docusaurus/docs/how-to-guides/ui-extensions/register-an-extension.md

```diff
@@ -21,6 +21,12 @@ Read more about extensions under [key concepts](../../key-concepts/ui-extensions
 
 <ExtensionPoints/>
 
+:::warning
+
+You must [update your plugin.json metadata](#updating-pluginjson-metadata) to list any registered extensions.
```

Comment:
```suggestion
You must [update your `plugin.json` metadata](#updating-pluginjson-metadata) to list any registered extensions.
```

Reply:
mixing monospace style in a hyperlink looks awful so I'll rework this instead